client/components/ProductOverview/ProductOverview.jsx

```diff
@@ -63,29 +73,49 @@ const ProductOverview = ({ productId }) => {
   }, []);
 
   return (
-    <Grid container spacing={2}>
-      <Grid item xs={6}>
-        <Box display="flex" flexDirection="column">
+    <>
+      <Grid container spacing={2}>
+        <Grid item xs={7}>
           <ImageGallery currentStyle={currentStyle} />
-        </Box>
-      </Grid>
-      <Grid container direction="column" justify="space-between" item xs={4}>
-        <Grid item>
-          <ProductInformation product={product} currentStyle={currentStyle} />
         </Grid>
-        <Grid item>
-          <StyleSelector
-            styles={styles}
-            currentStyle={currentStyle}
-            setCurrentStyle={setCurrentStyle}
-          />
+        <Grid container direction="column" justify="space-between" item xs={4}>
+          <Grid item>
+            <ProductInformation product={product} currentStyle={currentStyle} />
+          </Grid>
+          <Grid item>
+            <StyleSelector
+              styles={styles}
+              currentStyle={currentStyle}
+              setCurrentStyle={setCurrentStyle}
+            />
+          </Grid>
+          <Grid item>
+            <AddToCart currentStyle={currentStyle} />
+          </Grid>
+        </Grid>
+        <Grid item xs={1}></Grid>
+      </Grid>
+      <Grid container spacing={2}>
+        <Grid item xs={7} className={classes.description}>
+          <Typography variant="h6">{product.slogan}</Typography>
+          <Typography variant="body1">{product.description}</Typography>
         </Grid>
-        <Grid item>
-          <AddToCart currentStyle={currentStyle} />
+        {/* <Grid item xs={1} /> */}
+        <Grid item xs={5} className={classes.description}>
+          {product.features ? (
+            product.features.map((feature, idx) => (
+              <Typography key={idx}>
+                <CheckIcon /> {feature.value}
+                {'   '}
+                {feature.feature}
+              </Typography>
+            ))
+          ) : (
+            <></>
+          )}
```

Comment:
This CheckIcon seems handy.  I think I'll need that as well for my comparison modal!